nsnet2/index.html

```diff
@@ -60,7 +60,7 @@
                 Choose a sample audio
               </button>
               <div class="dropdown-menu">
-                <button class="dropdown-item" id="babbel">Babbel noise</button>
+                <button class="dropdown-item" id="babble">babble noise</button>
```

Comment:
I preprefer to use "Babble" as the button element content rather than "babble", especially other items are also capitalized. Keep the id name in lower case is reasonable. @NingW101